docs/content/reference/file-formats.md

```diff
@@ -158,70 +158,70 @@ status:
-| dependencies.requires.parameters | false    | A map of parameter names to their value.                                                                                                       |
-| customActions                    | false    | A map of action names to a custom action definition.                                                                                           |
-| customActions.NAME.description   | false    | A description of the action.                                                                                                                   |
-| customActions.NAME.modifies      | false    | Specifies if the action will modify resources in any way.                                                                                      |
-| customActions.NAME.stateless     | false    | Specifies that the action could be run before the bundle is installed and does not require credentials.                                        |
+| Field                            | Required | Description                                                                                                                                                                            |
+|----------------------------------|----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| schemaType                       | false    | The type of document. This isn't used by Porter but is included when Porter outputs the file, so that editors can determine the resource type.                                         |
+| schemaVersion                    | true     | The version of the Bundle schema used in this file.                                                                                                                                    |
+| name                             | true     | The name of the bundle.                                                                                                                                                                |
+| version                          | true     | The version of the bundle, must adhere to [semver v2].<br/>The bundle tag defaults to the version with a v prefix, e.g. mybundle:v1.0.0. Use --tag or the reference field to override. |
+| description                      | false    | A description of the bundle.                                                                                                                                                           |
+| registry                         | true*    | The OCI registry to use when the bundle is published.<br/> /*Either registry or reference must be specified.                                                                           |
```

Comment:
Wasn't sure if the `/*` prefix was intended (instead of `*`), but thought I'd mention/show it:

<img width="1013" alt="Screen Shot 2022-03-11 at 10 29 49 AM" src="https://user-images.githubusercontent.com/9322017/157897713-de0b3613-8751-4130-b098-6932814ef7d9.png">

Reply:
Oops, wrong direction for the slash. I was trying to escape the * in the markdown so that it would show up as an * and not a bullet point. Fixing!